lib/python/src/bailo/helper/entry.py

```diff
@@ -60,7 +69,7 @@ def card_from_schema(self, schema_id: str | None = None) -> None:
         res = self.client.model_card_from_schema(model_id=self.id, schema_id=schema_id)
         self.__unpack_card(res["card"])
 
-        logger.info(f"Card for ID %s successfully created using schema ID %s.", self.id, schema_id)
+        logger.info("Card for ID {self.id} successfully created using schema ID {schema_id}.")
```

Comment:
Could we change this log back to an fstring

Reply:
Done